app/views/rankings/show.html.erb

```diff
@@ -1,14 +1,37 @@
+<h1>Show Ranking</h1>
+
```

Comment:
spacing of lines on this page does not seem to be consistent with the rest of the application.